Formula/man-db.rb

```diff
@@ -0,0 +1,45 @@
+class ManDb < Formula
+  desc "Modern, featureful implementation of the Unix man page system"
+  homepage "https://nongnu.org/man-db/"
+  url "https://download.savannah.nongnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
```

Comment:
`man` is already in macOS, so this needs `keg_only :provided_by_macos`

Reply:
Uh, wouldn't that defeat the purpose of even *having* this formula? Git and Ruby are already in macOS too, but they're not keg-only...

Reply:
ruby is actually keg-only

Reply:
So it is. I think you know what I mean, though.